x/concentrated-liquidity/math/math.go

```diff
@@ -166,7 +166,7 @@ func GetNextSqrtPriceFromAmount0OutRoundingUp(sqrtPriceCurrent, liquidity, amoun
 // avoid overpaying out of the pool. Therefore, we round down.
 // sqrt_next = sqrt_cur + token_in / liq
 func GetNextSqrtPriceFromAmount1InRoundingDown(sqrtPriceCurrent, liquidity, amountOneRemainingIn osmomath.BigDec) (sqrtPriceNext osmomath.BigDec) {
-	return sqrtPriceCurrent.Add(amountOneRemainingIn.QuoTruncate(liquidity))
+	return sqrtPriceCurrent.Add(amountOneRemainingIn.QuoTruncateMut(liquidity))
```

Comment:
I don't think we can do this one. Can't mutate `amountOneRemainingIn`

Reply:
Yeah. u right, revert now